### DISCUSSION TITLE
Snapshot refresh in Home Assistant after upgrade to 0.14

### DISCUSSION THREAD

**Discussion by QuadrifoglioVerde** [blakeblackshear/frigate#13470] [Ask A Question]
### What is your question?

Hi, is anyone having the same problem with snaphosts refreshing in Home Assistant?
I have a camera with person detection, and the "person occupancy" value changes immediately, but the person image does not until after the detection ends. This does not happen in 0.13, there is a change of both immediately.

**Comment by PablaV** [comment]
take a look at this [thread](https://github.com/blakeblackshear/frigate/discussions/11696#discussion-6766663), there is a bug where Frigate sometime doesn't publish the snapshot via MQTT
